posLib/src/main/java/com/dedrick/poslib/POSPrinter.java

```diff
@@ -0,0 +1,338 @@
+package com.dedrick.poslib;
+
+import android.app.Application;
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.Canvas;
+import android.graphics.Rect;
+import android.util.Log;
+
+import com.newland.nsdk.esc.api.ESCPrinter;
+import com.newland.nsdk.esc.common.ESCPrinterException;
+import com.newland.nsdk.esc.internal.ESCPrinterImpl;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+public class POSPrinter {
+    Application app = null;
+    private ESCPrinter escPrinter = ESCPrinterImpl.getInstance();
+    private int myOtsu = 128;
+
+
+    POSPrinter(Application a){
+        this.app = a;
+
+    }
+    protected void print(String bank, String bankingAgent, String activityPoint, String address, String transactionType, String account_number,String stan, String created_at, String currency, String main_amount, String description, String terminalId){
```

Comment:
This is a super class, Newland is specializing it so put all Newland specific code to Newland.java and also rename Newland.java to Newland950.java